common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -105,7 +107,7 @@ object Mappings {
 
   val collectionMapping = withIndexName("collection", nonDynamicObj(
     "path" -> nonAnalysedList("collectionPath"),
-    "pathId" -> nonAnalyzedString,
+    "pathId" -> (nonAnalyzedString ++ copyTo("collections.pathHierarchy")),
```

Comment:
Is it worth documenting what the `copyTo` is for (iow, I don't know what it is for :smile_cat:)?